tests/scripts/all.sh

```diff
@@ -1065,10 +1065,9 @@ component_check_test_dependencies () {
     # the test code and that's probably the most convenient way of achieving
     # the test's goal.
     #
-    # Both MBEDTLS_ASN1_[PARSE|WRITE]_C are used in ECDSA conversion functions
+    # MBEDTLS_ASN1_WRITE_C is also used in ECDSA conversion functions
```

Comment:
I don't think that's the case any more, so I think this line and the next could be removed - keeping this whole comment the way it was before this PR.